### PR TITLE
Update README.md for Sphinx documentation

### LINKED ISSUE #9: Update README.md for Sphinx documentation
With docs published on ReadTheDocs, here are a few necessary changes to minimize docs maintainence.

- [x] Make the README.md concise. Only keep the most important item
- [x] Have a TOC that points to separate sections of the ReadTheDocs pages
- [x] Include GitHub badges for
   - [x] RTDs build status
   - [x] GitHub Actions Sphinx build status

**Notes**

RTD Build status badges cannot be fully customized. This means that (1) the label will always be called `docs`, and that (2) the color will always reflect the build status (green, red, etc.). In fewer words: we cannot make the RTDs badge style match the other ReDATA badges.

For documentation see: 
https://docs.readthedocs.io/en/stable/badges.html#status-badges
and https://raw.githubusercontent.com/readthedocs/readthedocs.org/master/docs/badges.rst

**Implemented in:** docs/09_README

### PR DESCRIPTION
**Pull request by damian-romero**
Fix #9 

To be tagged as `v0.3.1`
- [x] Update `setup.py`
- [x] Update `__init__.py`

Contains separate commits for: 

- Adding badges
- Updating table of contents
- Simplification of README.md contents

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,21 @@
...
-
-code_path = "/path/to/repo"
-gi = redata.commons.git_info.GitInfo(code_path)
-```
-
-or
-
-```python3
-from redata.commons.git_info import GitInfo
+- [Overview]([#overview](https://redata-commons.readthedocs.io/en/latest/#overview))
```
Comment (astrochun) [inline README.md:10]: The Overview link does not work. It's incorrectly formatted

File: README.md [hunk 2]
```diff
@@ -1,57 +1,21 @@
...
+- [Execution](https://redata-commons.readthedocs.io/en/latest/execution.html#execution)
+    - [Using `git_info`](https://redata-commons.readthedocs.io/en/latest/execution.html#using-git-info)
+    - [Using `logger`](https://redata-commons.readthedocs.io/en/latest/execution.html#using-logger)
+- [Authors](#authors)
+- [License](#license)
 
-code_path = "/path/to/repo"
-gi = GitInfo(code_path)
-```
+This repository contains commonly used codes by ReDATA software.
```
Comment (astrochun) [inline README.md:18]: I would move this above TOC. It should not be this far down.

File: README.md [hunk 3]
```diff
@@ -1,57 +1,21 @@
...
-
-```python3
-from redata.commons.git_info import GitInfo
+- [Overview]([#overview](https://redata-commons.readthedocs.io/en/latest/#overview))
+- [Installation](https://redata-commons.readthedocs.io/en/latest/installation.html#installation)
+- [Execution](https://redata-commons.readthedocs.io/en/latest/execution.html#execution)
+    - [Using `git_info`](https://redata-commons.readthedocs.io/en/latest/execution.html#using-git-info)
+    - [Using `logger`](https://redata-commons.readthedocs.io/en/latest/execution.html#using-logger)
+- [Authors](#authors)
+- [License](#license)
```
Comment (astrochun) [inline README.md:16]: I wonder if we should just link to RTDs here. Although having it both in the code and RTDs might make sense for completeness (both Authors and License are part of code metadata).

I would also add a link to "API Documentation"

**Review by astrochun — CHANGES_REQUESTED** [review]
Overall good. A few suggestions.

File: README.md (hunk 1 above)
Comment (damian-romero) [reply README.md:10]: Thanks, good catch.

File: README.md (hunk 2 above)
Comment (damian-romero) [reply README.md:18]: Thanks. I was wondering if that should be the case.

File: README.md (hunk 3 above)
Comment (astrochun) [reply README.md:16]: We discussed and decided to keep an extra section for developers to find.

Comment (damian-romero) [reply README.md:16]: - Deleted Authors' list writeup and added link to RTDs instead
- Kept  Licence in README.md as per our discussion
- Added link to API Documentation

**Review by astrochun — APPROVED** [review]

**Review by astrochun — APPROVED** [review]
Looks good.

**Pull request merged**